app/javascript/components/subject/CaseStatus.js

```diff
@@ -167,8 +167,7 @@ class CaseStatus extends React.Component {
         <Modal.Body>
           {this.state.confirmedOrProbable &&
             this.props.patient.monitoring &&
-            this.props.patient.case_status !== 'Confirmed' &&
-            this.props.patient.case_status !== 'Probable' && (
+            ((this.props.patient.case_status !== 'Confirmed' && this.props.patient.case_status !== 'Probable') || !this.state.isolation) && (
```

Comment:
This did fix the issue.  I'm thinking this logic should be encapsulated with a `showMonitoringDropdown` state variable that is also set in `handleCaseStatusChange.` Would make it easier to find the logic for each possible case too. Thoughts?

Reply:
yup I agree - it can be set within this case and should be good to go

```
        // changing case status to Confirmed or Probable (excluding case directly above)
      } else if (confirmedOrProbable) {
        this.setState({ disabled: true });
```

Reply:
done